administration/config/webpack.config.ts

```diff
@@ -518,6 +518,7 @@ function createWebpackConfig(webpackEnv: 'development' | 'production'): webpack.
             syntactic: true,
           },
           mode: 'write-references',
+          build: true,
```

Comment:
Do you know what this does? I am not entirely sure :D

I am wondering:
Maybe we want to remove TS checking completely from the webpack build and directly transpile to JS to improve build times?
Type checking happens anyway in the language server in the IDE as well as on CI? Similarly, we don't check eslint / stylelint when building (one could see TS as a linter on steroids :D )

Reply:
> Do you know what this does? I am not entirely sure :D

This runs tsc --build before typechecking, which will generate the missing d.ts files.

But I think you're right. I added a typescript run to the CI, as it was missing and removed typechecking during build. @f1sh1918 it should work now even without the symlink. Would you mind giving it a try? :)